### PR TITLE
Get ready for v0.11 and v1.0

### PR DESCRIPTION
**Pull request by hiroyuki-sato**
This PR catchup [Get ready for v0.11 and v1.0!](https://dev.embulk.org/topics/get-ready-for-v0.11-and-v1.0.html)

v0.10 API ready and also, still supports v0.9.

@dmikurube Could you take a look when you get a chance?

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,36 +1,61 @@
...
-    id "com.github.jruby-gradle.base" version "0.1.5"
     id "java"
     id "checkstyle"
+    id "maven-publish"
+    id "org.embulk.embulk-plugins" version "0.4.2"
 }
-import com.github.jrubygradle.JRubyExec
 repositories {
     mavenCentral()
     jcenter()
```
Comment (dmikurube) [inline build.gradle]: We no longer need jcenter?

**Review by dmikurube — APPROVED** [review]
LGTM! Left one.comment.

**Comment by hiroyuki-sato** [comment]
@dmikurube Thanks!

@civitaspo Could you take a look when you get a chance?

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Wed Jan 13 12:41:02 JST 2016
+#Mon May 03 22:48:55 JST 2021
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.10-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.3-bin.zip
```
Comment (civitaspo) [inline gradle/wrapper/gradle-wrapper.properties]: @hiroyuki-sato Run `./gradlew wrapper` and then commit updated files like `./gradlew`, `./gradlew.bat`.

**Review by civitaspo — COMMENTED** [review]
@hiroyuki-sato Thank you for the contribution. This is very helpful for updating my Embulk plugin.

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (hiroyuki-sato) [reply gradle/wrapper/gradle-wrapper.properties]: Thanks! Hmm
a12f39bc1279484ea4384a2348cbe0f737a2d892 
I executed `./gradlew wrapper --gradle-version=6.8.3` I'm not sure why to execute `./gradlew wrapper` again. Anyway, I executed it.

**Comment by hiroyuki-sato** [comment]
@civitaspo Thank you for your comment. I fixed your comment. Please take a look again.

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (dmikurube) [reply gradle/wrapper/gradle-wrapper.properties]: Jfyi, the first `./gradle wrapper` only updates this file. Then the second `./gradlew wrapper` is sometimes needed to download new wrapper start-up scripts `gradlew` and `gradlew.bat`.

Comment (hiroyuki-sato) [reply gradle/wrapper/gradle-wrapper.properties]: Thanks! I tried to update the `embulk-parser-jsonpath` plugin at first.
The gradle version was 6.2.
When I execute `./gradlew wrapper --gradle-version=6.8.3`. It updates four files.
(`gradle/wrapper/gradle-wrapper.jar`, `gradle/wrapper/gradle-wrapper.properties`, `gradlew` and `gradlew.bat`)

The gradle version of this project is `2.10`. It doesn't change those files. I misunderstood. 😄 


```
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   gradle/wrapper/gradle-wrapper.jar
	modified:   gradle/wrapper/gradle-wrapper.properties
	modified:   gradlew
	modified:   gradlew.bat
```

**Review by civitaspo — APPROVED** [review]

**Comment by civitaspo** [comment]
https://github.com/civitaspo/embulk-filter-to_json/releases/tag/v0.1.0

**Pull request merged**